src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -229,17 +226,12 @@ public void testPostUpdateUserInfo() throws Exception {
     servletUnderTest.doPostWrapper(datastore, mockRequest, mockResponse);
 
     // Query the local datastore for the newly created user entity
-    PreparedQuery results = datastore.prepare(new Query(TEST_USER_ID));
-    Assert.assertEquals(1, results.countEntities());
-    Entity userEntity = results.asSingleEntity();
+    Entity userEntity = datastore.prepare(new Query(TEST_USER_ID)).asSingleEntity();
+    assertThat(userEntity).isNotNull();
 
     // Verify that the user Entity name property changed
-    Assert.assertEquals((String) userEntity.getProperty(USER_ID_PROPERTY), TEST_USER_ID);
-    Assert.assertEquals((String) userEntity.getProperty(USER_NAME_PROPERTY), ALTERNATE_TEST_USER_NAME);
-    Assert.assertEquals((String) userEntity.getProperty(USER_EMAIL_PROPERTY), TEST_USER_EMAIL);
-    Assert.assertEquals((String) userEntity.getProperty(USER_BIO_PROPERTY), TEST_USER_BIO);
-    Assert.assertEquals(((ArrayList<String>) userEntity.getProperty(USER_FRIENDS_LIST_PROPERTY)).toArray(),
-        TEST_USER_FRIENDS_LIST);
+    assertThat((String) userEntity.getProperty(USER_ID_PROPERTY)).contains(TEST_USER_ID);
```

Comment:
`.isEqualTo()` - `contains()` (and it's cousins) are for collections. This is a string and you are looking for equality.

Reply:
That makes sense.